product_docs/docs/biganimal/release/getting_started/preparing_cloud_account/01_preparing_azure/01_understanding_qotas_in_azure.mdx

```diff
@@ -27,6 +27,27 @@ To prevent failures while creating your clusters, ensure that each of the follow
 | Microsoft.OperationsManagement | Monitors workloads and provides container insight                                                      |
 | Microsoft.Portal               | Provides a dashboard to monitor the running status of the clusters (using aggregated logs and metrics) |
 
+## Regional activation resources and access requirements
+
+The table provides BigAnimal's regional activation resources and access requirements.
+
+| Resource type                        | Activated region | Network access | Description                                                          |
+|--------------------------------------|------------------|----------------|----------------------------------------------------------------------|
+| Virtual network                      | Yes              | N/A            |                                                                      |
+| Storage account for backup           | yes              | Public         |                                                                      |
+| KeyVault for backup, AKS             | Yes              | Public         |                                                                      |
+| Storage account for logs and metrics | Yes              | Private        | Public is required for faraway replication and cross region restore. |
+| KeyVault for logs and metrics        | Yes              | Private        |                                                                      |
+| Managed Identify                     | Yes              | N/A            |                                                                      |
+| NSG                                  | Yes              | N/A            |                                                                      |
+| Private endpoint                     | Yes              | N/A            |                                                                      |
+| AKS                                  | Yes              | N/A            |                                                                      |
+| Application insights                 | Yes              | N/A            | Not in use for custom monitoring                                     |
+| Log analytics workspaces             | Yes              | N/A            | Not in use for custom monitoring                                     |
+
+!!!note
+A Storage account for TFstate is created on the first activated region per BigAnimal project. It is in private network access and the private DNS zone gets created.
```

Comment:
Can we change the note as:
A Storage account for TFstate is created on the first activated region per BigAnimal project. It is in private network access. Private DNS zones are required for private Storage Account and private Key Vaults.